webpack.config.js

```diff
@@ -8,9 +8,7 @@ const OptimizeCssAssetsPlugin = require('optimize-css-assets-webpack-plugin');
 const webpack = require('webpack');
 
 module.exports = {
-  entry: {
-    app: resolve(__dirname, 'src/client/index.js'),
-  },
+  entry: ['@babel/polyfill', './src/client/index.js'],
```

Comment:
Could you please talk more about this particular change?

Reply:
Is to Because fix regeneratorRuntime errors
Please visit [This web site](https://babeljs.io/docs/en/babel-polyfill) for more info